google/cloud/bigtable/version.h

```diff
@@ -38,11 +38,9 @@
 
 namespace google {
 namespace cloud {
-/**
- * Contains all the Cloud Bigtable C++ client APIs.
- */
```

Comment:
I kind of like this comment. It shows up in the dox: https://screenshot.googleplex.com/3rin8yunN3qzskp

Reply:
Restored here, there, and everywhere.